feat: add mapped version description (#16)
seri037/llcppg: llpkgstore.md

### PR verification workflow
1. Ensure that there is only one `llpkg.cfg` file across all directories. If multiple instances of `llpkg.cfg` are detected, the PR will be aborted.  
2. Check if the directory name is valid, the directory name in PR **SHOULD** equal to `Package.Name` field in the `llpkg.cfg` file.
3. Check the PR commit footer contains a {MappedVersion}.

### llpkg generation

A standard method for generating valid llpkgs:
1. Receive binaries/headers from [installer](#llpkgcfg-structure), and index them into `.pc` files
2. Automatically generate llpkg using a [generator](#llpkgcfg-structure) for different platforms
3. Combine generated results into one Go module
4. Debug and re-generate llpkg by modifying the configuration file

### Version tag rule
1. Parse the `{MappedVersion}` of current package from PR commit footer
2. Follow Go's version management for nested modules. Tag `{CLibraryName}/{MappedVersion}` for each version.
3. This design is fully compatible with native Go modules
    ```
    github.com/goplus/llpkg/cjson@v1.7.18
    ```

### `{MappedVersion}` in PR commit

> `{MappedVersion}` **MUST** be included in the footer of the latest commit in the current PR and must follow this format:  

Reviewer: ```suggestion
`{MappedVersion}` **MUST** be included in the PR's latest commit, and **MUST** follow the format:  
```

```
Commit-as: {MappedVersion}
```  

The PR verification process will validate this format and abort the PR if it is invalid.

Example:
```
git commit -m "feat: add cjson" -m "Commit-as: v1.0.0"
git merge
```

> 

Reviewer: ```suggestion
```
